doc/api/globals.md

```diff
@@ -637,6 +637,51 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
+
+The value is representing the language version as defined in [RFC 5646][].
+
+You can override the value of `navigator.language` by setting the
+`LC_ALL` environment variable.
+
+For example:
+
+```sh
+LC_ALL=fr-FR node -p "console.log(navigator.language)"
```

Comment:
```suggestion
LC_ALL=fr-FR node --print "console.log(navigator.language)" // 'fr-FR'
```

In general let’s use long flag names so that people don’t need to look up what the short flags mean.